waku/v2/node/waku_node.nim

```diff
@@ -172,6 +178,11 @@ proc new*(T: type WakuNode,
 
   info "Initializing networking", addrs= $netConfig.announcedAddresses
 
+  let relayClient = RelayClient.new()
+  let autoRelayService = AutoRelayService.new(1, relayClient, nil, rng)
+  let autonatService = getAutonatService(rng)
+  let hpservice = HPService.new(autonatService, autoRelayService)
```

Comment:
wondering if we should have hole punching behind a flag or always on by default. specially now at the begining that its new. opinion? @jm-clius

Reply:
Maybe a flag that is true by default